kubetest/main.go

```diff
@@ -114,6 +114,10 @@ type options struct {
 	testCmdArgs        []string
 	up                 bool
 	upgradeArgs        string
+	// sshProxyInstanceName is the name of the vm instance which ip address will be used to set the
+	// KUBE_SSH_BASTION env. If set, it will result in proxying ssh connections in tests through the
+	// "bastion". It's useful for private cluster where nodes don't have public ip addresses.
```

Comment:
Private clusters is more gke-specific term. Maybe sth like:
It's useful for cluster with node without ssh access exposed to the whole world (e.g. nodes without public ip addresses)?